config/locales/en/errors.yml

```diff
@@ -29,6 +29,11 @@ en:
       heading: Access restricted
       lead_text: This page requires you to be signed in to LAA Portal. You will be redirected back to this service after you sign in.
       sign_in_button: Sign in with LAA Portal
+    datastore:
```

Comment:
It feels like this might be best to place in the existing `dashboard.yml` maybe under `en.shared.errors`?

These other errors in this locale are full page errors, each one being a routable URL.